Dockerfile

```diff
@@ -0,0 +1,38 @@
+# base image
+FROM python:slim-buster as base
+# update Image
+RUN apt-get update
+# install poetry
+RUN pip install poetry
+# copy all except in dockerignore
+COPY . .
+# install prerequisits
+RUN poetry install --no-root --no-dev
+RUN pip install -r requirements.txt
```

Comment:
The poetry install should be enough, shouldn't need pip

Reply:
i hashed out line 6 Run pip install poerty but then got an error on Run poetry install --no-root --no-dev

Reply:
so i have left this as is

Reply:
Ah sorry, you will still need to install poetry (line 6), but poetry should take care of the rest of the dependencies, so you shouldn't need line 11